.github/ISSUE_TEMPLATE/E-proponer-software-para-revision.md

```diff
@@ -6,7 +6,8 @@ about: ¿Quieres confirmar que tu paquete encaja para revisión? Usa esta planti
 
 Persona Encargada: nombre (@github_handle)  
 Otras Autoras del Paquete: (borra si solo hay una autora) nombre (@github_handle)
-Repositorio:   
+Repositorio: <!--repourl--><!--end-repourl-->
+Typo de Envio: <!--submission-type-->pre-Envio<!--end-submission-type-->
```

Comment:
why small p and big E?

Reply:
German-Spanish-English capitalization confusion ... :confused: ... Small `p` because it's small on the English one. Shall we small-e as well?